examples/closures/closures.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+
+int globalInt = 5;
+
+//basic example where a lambda with a if block access its outer scope variables
```

Comment:
There should be a space after "//", and also the comment should start with a capital letter.